README.md

```diff
@@ -108,8 +108,24 @@ tests/              - test cases package
     "message": "Invalid Data Type"
 }
 ```
+### DELETE /recommendations
 
+##### Request Parameter
+DELETE /recommendations/{id}
 
+##### Headers
+- Content-Type: application/json
+
+##### Response
+- Status: 204 No Content
+
+- Status: 404 Not found
```

Comment:
Remove status 404, and it's json. We should only return 204 everytime, and not 404. (We have done correctly in the `routes.py` delete api code, so just document accordingly)

Reply:
Removed it.